docs/product-installation-guide/src/main/asciidoc/eap-users-create-proc.adoc

```diff
@@ -1,29 +1,29 @@
 [id='eap-users-create-proc']
 
 = Creating users
-Before you can use {CENTRAL}, you must create a user that has the `admin` role. To log in to {KIE_SERVER}, you must create a user that has the `kie-server` role. You can create a single user that has both of these roles.
+Before you can log in to {CENTRAL} or {KIE_SERVER}, you must create users. This section shows you how to create a {CENTRAL} user with the `admin` and `rest-all` roles and a {KIE_SERVER} user that has the `kie-server` role. For information about roles, see <<roles-users-con>>.
```

Comment:
There is one thing which should be incorporated. If Business Central user wants to perform any runtime operation in Business central, then he needs to have also kie-server role (because runtime operations on Business Central are performed using connected Kie servers)